tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -235,16 +235,16 @@ class SelectorProtocolPart(ProtocolPart):
 
     name = "select"
 
-    def element_by_selector(self, selector):
-        elements = self.elements_by_selector(selector)
+    def element_by_selector(self, element_selector, frame="window"):
+        elements = self.elements_by_selector(element_selector, frame)
         if len(elements) == 0:
-            raise ValueError("Selector '%s' matches no elements" % selector)
+            raise ValueError("Selector '%s' matches no elements" % element_selector)
```

Comment:
nit: maybe expand this error to specify the frame that we are selecting in? I think it would help debugging.
Also the error message below.